lib/lotus/routing/http_router.rb

```diff
@@ -112,6 +115,18 @@ def raw_call(env, &blk)
         super(@parsers.call(env))
       end
 
+      # Resolve the given Rack env to a registered endpoint and invoke it.
+      #
+      # @param env [Hash] a Rack env instance
+      #
+      # @return [Rack::Response, Array]
+      #
+      # @since x.x.x
+      # @api private
+      def call(env)
```

Comment:
@AlfonsoUceda Can you please make this working in `#raw_call` instead of here? `http_router` does low level stuff with this method (undefining and redefining it), that's why `#raw_call` it's there.